.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: گزارش باگ
+labels: bug
+---
+
+## شرح باگ
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)